gitlab/v4/objects/iterations.py

```diff
@@ -16,11 +16,31 @@ class GroupIterationManager(ListMixin, RESTManager):
     _path = "/groups/{group_id}/iterations"
     _obj_cls = GroupIteration
     _from_parent_attrs = {"group_id": "id"}
-    _list_filters = ("state", "search", "include_ancestors")
+    # The "in" keyword collides with python's "in" keyword raising a SyntaxError.
```

Comment:
I should probably clarify this - this is only an issue with direct API usage. We supply the `in` argument differently to the CLI. So this actually works for the CLI and we can add the `in` keyword here. Sorry for the confusion.

```suggestion
    # When using the API, the "in" keyword collides with python's "in" keyword raising a SyntaxError.
```